compiler/optimizer/Inliner.cpp

```diff
@@ -493,12 +493,6 @@ TR_InlinerBase::cleanup(TR::ResolvedMethodSymbol * callerSymbol, bool inlinedSit
 
    }
 
-bool
-OMR_InlinerPolicy::mustBeInlinedEvenInDebug(TR_ResolvedMethod * calleeMethod, TR::TreeTop *callNodeTreeTop)
```

Comment:
This one is used by downstream projects and can't be deleted.

Reply:
commit added per your comments @cathyzhyi 
4c3f9bf71d5f963d6d8455e56a554b5035df3023